lightning/src/chain/channelmonitor.rs

```diff
@@ -3441,7 +3468,7 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 				}
 				self.is_resolving_htlc_output(&tx, height, &block_hash, &logger);
 
-				self.is_paying_spendable_output(&tx, height, &block_hash, &logger);
+				self.check_tx_and_push_spendable_outputs(&tx, height, &block_hash, &logger);
```

Comment:
If a counterparty is paying to one of the spendable descriptor (e.g our `payment_point` as a P2WSH or P2WPKH) from let’s say a second-stage malleable HTLC-transaction on our local / holder commitment transaction, we might receive a descriptor of a “tag” value that can be latter used for e.g deanonymization attack (at the image of sending dust value to on-chain wallet to fingerprint them).

I don’t think this is a new concern and this sounds already permissible by current `is_paying_spendable_output`. l still think our transaction pattern matching in `transactions_confirmed` could be stronger to check that non-commitment-or-closing-transaction-paying-to-us-is-spent-by-us.